### PR TITLE
Migrate to AndroidX

### PR DESCRIPTION
**Pull request by slavkoder**
- [x] Update to use AndroidX artifacts
- [x] Update Gradle plugin.
- [x] Modify [corresponding guide](https://github.com/codepath/android_guides/wiki/Fragment-Navigation-Drawer/_compare/12e052269c60fc08532f4cca63c3e6084372fea6...7c15e29b08ff05d059f76f2c66052384884e54e3)
- [x] Verify guide code. I had to actually implement part of this lab to verify and update info in the guide ([branch](https://github.com/codepath/android-navigation-drawer-exercise/tree/slava/verify-navigation-drawer)).

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -2,6 +2,7 @@ apply plugin: 'com.android.application'
 
 android {
     compileSdkVersion 28
+    buildToolsVersion '29.0.2'
```
Comment (rogerhu) [inline app/build.gradle]: I think you can remove and Gradle will auto dl...

File: .travis.yml [hunk 2]
```diff
@@ -7,6 +7,7 @@ android:
   - tools
   - platform-tools
   - android-28
+  - build-tools-29.0.2
```
Comment (rogerhu) [inline .travis.yml]: Remove in both places?

Comment (slavkoder) [reply .travis.yml]: Travis was failing to build with `Warning: License for package Android SDK Build-Tools 29.0.2 not accepted.`, so I added this dependency here. I think I can remove it from `build.gradle` though.

**Comment by slavkoder** [comment]
@rogerhu works now, let me know if I'm missing something.

**Pull request merged**